README.md

```diff
@@ -46,7 +46,12 @@ When first creating your board repository, and every time the Pull Request Templ
     * Datasheet/Mouser link
     * Footprint Association
 7. If adding footprint, click on `Footprint Editor`, select a `UTSVT` folder to add it to.
-8. If component takes up vertical or horizontal space (buck converters, connectors, large capacitors, etc.), click on `Edit Foootprint Properties` -> `3D Models` and add 3d model association with footprint.
+8. If component takes up vertical or horizontal space (buck converters, connectors, large capacitors, etc.), follow the below steps:
+    1. Download footprint and store in THIS REPOSITORY - NOT THROUGH THE SUBMODULE. This means you must clone this repository and store the file in `UTSVT-KiCADLibraries/3DModels`
+    2. Open KiCAD through your project file in your PCB repository. Select `Preferences` -> `Configure File Paths` -> `+` -> and add path to 3D Models (Starting from C: drive) as an environment variable called `UTSVT-3DModels`.
+    3. Now when adding the 3D Model to the footprint, just select the footprint through the footprint library, click on 3D Models, and add the name of the step file.
+
+    **NOTE**: The way this works is the environment variable is configured for your KiCAD application on just your machine. This means even if you go through these steps, someone else who clones your repository will still have to go through the same steps to ensure that they have access to the 3D Models.
```

Comment:
IDK about windows, but the KIPRJMOD path variable is autogenerated from the kicad project; it can be used to have user agnostic paths to symbols, footprints, and 3d models.
3D model paths will have to be added manually through the footprint editor.

Reply:
![image](https://github.com/lhr-solar/UTSVT-KiCadLibraries/assets/31521956/41e23c56-633a-4941-b0b0-d9978dcf4e7a)

Reply:
I tried, but that is just for symbol and footprint libraries. There are no options for path substitutions for 3D Models.

Reply:
Ok then, I will change the instructions

Reply:
seconded
![image](https://github.com/lhr-solar/UTSVT-KiCadLibraries/assets/31521956/610f766e-cc17-4291-aa10-ebf6e877447a)